client/src/components/App.jsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import Details from './Details';
-import { Map, GoogleApiWrapper } from 'google-maps-react';
+import $ from 'jquery';
+import { Map } from 'google-maps-react';
 import NeighborHoodMap from './NeighborhoodMap';
+import '../../dist/index.css';
```

Comment:
Please modularize into separate CSS modules in a subsequent PR. I won't block you on this one here though.